packages/lesswrong/components/posts/LWPostsItem.tsx

```diff
@@ -370,6 +370,7 @@ const LWPostsItem = ({classes, ...props}: PostsList2Props) => {
     curatedIconLeft,
     strikethroughTitle,
     bookmark,
+    className,
```

Comment:
This is unused at the moment, but the signature for `EAPostsItem` and `LWPostsItem` needs to match.